kore/src/main/scala/org/kframework/builtin/Sorts.scala

```diff
@@ -7,6 +7,7 @@ import org.kframework.kore.KORE.Sort
 
 object Sorts {
   val Layout = Sort("#Layout")
+  val LineMarker = Sort("#LineMarker")
```

Comment:
I don't see `#LineMarker` documented in https://kframework.org/USER_MANUAL/. Can you add a blurb there about it (maybe in the same place that `#Layout` is described).